install/syndesis.yml

```diff
@@ -469,6 +469,9 @@ objects:
           "logging": false
         },
         "branding": {
+          "logoWhiteBg": "",
+          "logoDarkBg": "",
+          "iconWhiteBg": "assets/images/FuseOnlineLogo_Black.svg",
```

Comment:
This change looks a bit unrelated.

Reply:
This is a derived resource. Previous commit(s) didn't regenerate it.